lib/auth.ts

```diff
@@ -196,14 +196,15 @@ export class Auth {
       HttpMethod.GET
     );
```

Comment:
Good catch, you are totally right of course 🙂I think I like rejecting a little bit more in this case because otherwise the user could be stuck while multiple requests are failing without being notified of it. Rejecting allows a clean "try again later" approach. What do you think?

I have changed here: bcb4212999739c49099d17771c0946e760b653aa

Reply:
I think the problem with internet is very rare on desktop, so it would be fine to just reject because most likely something is wrong with the server 👍